data/anaconda.conf

```diff
@@ -267,6 +267,11 @@ hidden_spokes =
 # Run GUI installer in a decorated window.
 decorated_window = False
 
+# do not have hard requirement on root password being set by default
```

Comment:
I don't understand the description. Maybe `Require the root password to be set.` might be better?